watercore-base/src/main/java/me/lucyy/watercore/core/WaterCorePlugin.java

```diff
@@ -34,6 +34,9 @@
 
 	@Override
 	public void onEnable() {
+
+		if(getVersion() > 11) getServer().getPluginManager().disablePlugin(this);
```

Comment:
Please add braces and a descriptive console error

Reply:
Is this good for console message:
`[WaterCore] WaterCore requires Java 11 or higher! Disabling plugin.`

Reply:
Yes that'd be fine, just don't put the prefix in, bukkit does that for you